pkg/apis/core/v1/storagecluster.go

```diff
@@ -300,6 +300,12 @@ type RollingUpdateStorageCluster struct {
 	// that at least 70% of original number of StorageCluster pods are available at
 	// all times during the update.
 	MaxUnavailable *intstr.IntOrString `json:"maxUnavailable,omitempty"`
+
+	// Minimum number of seconds for which a newly created Portworx pod should be ready
+	// without any of its container crashing, for it to be considered available.
+	// Defaults to 0 (pod will be considered available as soon as it is ready)
+	// +optional
+	MinReadySeconds int32 `json:"minReadySeconds,omitempty"`
```

Comment:
could you run `make codegen` there should be deep copy function changes for this

Reply:
Good point, i have pushed a new commit.